.github/workflows/build.yaml

```diff
@@ -98,7 +98,25 @@ jobs:
         with:
           name: pages
           path: dist/design
-# TODO: build design system
+
+  publish-documentation:
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-22.04
+    permissions:
+      pages: write
+      id-token: write
+    needs:
+      - documentation
+    steps:
+      - uses: actions/configure-pages@v5
+      - uses: actions/download-artifact@v4
+        with:
+          name: pages
+          path: dist/design
+      - uses: actions/upload-pages-artifact@v3
+        with:
+          path: 'dist/design'
+      - uses: actions/deploy-pages@v4
```

Comment:
Probably more of a follow-up topic, but we might want to define an envrionment here at some point: https://github.com/actions/deploy-pages?tab=readme-ov-file#usage.

Reply:
yes, once we can :laughing: